src/components/icons/index.tsx

```diff
@@ -1,2 +1,2 @@
 export * from "./Icon";
-export { Icon, AnimatedIcon } from "../icons/Icon";
+export { Icon, AnimatedIcon } from "./Icon";
```

Comment:
Is this needed? we already export all from `./Icon` in the previous line
```suggestion
```

Reply:
Yes, I'll remove the line.